README.md

```diff
@@ -15,13 +15,17 @@ Jenkins X also participates in GSoC under the Jenkins umbrella.
 If you are a project maintainer and consider mentoring during the GSoC 2020 cycle, please, submit your ideas below using the [template](PROJECT_IDEA_TEMPLATE.md).
 
 ### Jenkins
+Jenkins is the leading open source automation source built on the Java Virtual Machine (JVM) with an ecosystem of >2000 plugins that extend Jenkins to automate with practically any technology software delivery teams use.
```

Comment:
This `Jenkins is the leading open source automation source built` feels like it should be `Jenkins is the leading open source automation tool built` using source twice seems incorrect? Is it just me?

Reply:
good catch - it should be server - fixing...